Clean up FAQs sections and add new Get Started FAQs (#219)
OffchainLabs/arbitrum-docs: arbitrum-docs/partials/_troubleshooting-users-partial.md

### Why do I need ETH to use the Arbitrum network? {#why-do-i-need-eth-to-use-the-arbitrum-network}

> <p>When moving funds from Mainnet (L1) to Arbitrum (L2), you'll need ETH in your Arbitrum wallet. Even if you want only to use another, non-ETH, token, you will still need a small amount of ETH in your wallet on the Arbitrum network. This is because all Arbitrum transactions are powered by ETH. ETH is the currency used for gas fees.</p>

Reviewer: We use the word **Ethereum** in other parts of the docs. Should we rename **Mainnet** to **Ethereum** for the sake of consistency?

Reviewer: Also I'd just an small rewording here as the **Arbitrum Walle**t might sounds confusing:
```suggestion
<p>When moving funds (ETH and non-ETH) from Mainnet (L1) to Arbitrum (L2), you'll need to have ETH in your wallet on the corresponding Arbitrum chain. This is because ETH is the currency used for gas fees on Arbitrum and  all Arbitrum transactions are powered by ETH.</p>
```

<p></p>

<p></p>

### Do I need to pay a tip / Priority fee for my Arbitrum transactions? {#do-i-need-to-pay-a-tip--priority-fee-for-my-arbitrum-transactions}

> <p>Since transactions are processed in the order that the Sequencer receives them, no priority fee is necessary for Arbitrum transactions; if a transaction does include a priority fee, it will be refunded to the transaction's origin address at the end of execution.</p>

Reviewer: ```suggestion
<p>Since transactions are processed in the order that the Sequencer receives them, no priority fee is necessary for Arbitrum transactions; if a transaction does include a priority fee, it will be refunded to the transaction's origin address at the end of the execution.</p>
```

> ### How can I see the balance of my Eth / Tokens on Arbitrum in my wallet? {#how-can-i-see-the-balance-of-my-eth--tokens-on-arbitrum-in-my-wallet}

Reviewer: ```suggestion
### How can I see the balance of my ETH / Tokens on Arbitrum in my wallet? {#how-can-i-see-the-balance-of-my-eth--tokens-on-arbitrum-in-my-wallet}
```

> ### If there is a dispute, can my L2 transaction get reorged / throw out / "yeeted"? {#if-there-is-a-dispute-can-my-l2-transaction-get-reorged--throw-out--yeeted}

Reviewer: ```suggestion
### If there is a dispute, can my L2 transaction get reorged / thrown out / "yeeted"? {#if-there-is-a-dispute-can-my-l2-transaction-get-reorged--throw-out--yeeted}
```

<p>Nope; once an Arbitrum transaction is included on L1, there is no way it can be reorged (unless the L1 itself reorgs, of course). A "dispute" involves Validators disagreeing over execution, i.e., the outputted state of a chain. The inputs, however, can't be disputed; they are determined by the Inbox on L1. (See <a href="https://developer.arbitrum.io/tx-lifecycle">Transaction Lifecycle</a>)</p>

### ...okay but if there's a dispute, will my transaction get delayed?<a href="https://developer.arbitrum.io/faqs/protocol-faqs#q-dispute-delay"></a>
<p>The only thing that a dispute can add delay to is the confirmation of L2-to-L1 messages. All other transactions continue to be processed, even while a dispute is still undergoing. (Additionally: in practice, most L2-to-L1 messages represent withdrawals of fungible assets; these can be trustlessly completed <em>even during a dispute</em> via trustless fast "liquidity exit" applications. See <a href="https://developer.arbitrum.io/arbos/l2-to-l1-messaging">L2-to-L1 Messages</a>).</p>

## 

> ### Are "Sequencers" the same thing as "Validators"? Can a centralized Sequencer do bad things like steal all my money? {#are-sequencers-the-same-thing-as-validators-can-a-centralized-sequencer-do-bad-things-like-steal-all-my-money}

Reviewer: ```suggestion
### Are "Sequencers" the same thing as "Validators"? Can a centralized Sequencer act maliciously (e.g., steal all my money)? {#are-sequencers-the-same-thing-as-validators-can-a-centralized-sequencer-do-bad-things-like-steal-all-my-money}
```

<p>No and no!</p>

<p>An Arbitrum Chain's Sequencer(s) and Validators and completely distinct entities, with their own distinct roles.</p>

> <p>The <a href="https://developer.arbitrum.io/sequencer">Sequencer</a> is the entity granted specific privileges over ordering transactions; once the Sequencer commits to an ordering (by posting a batch on Ethereum), it has no say over what happens next (i.e., execution). A malicious/faulty Sequencer can do things like reorder transactions or <em>temporarily</em> delay a transaction's inclusion — things which could be, to be sure, annoying and bad — but can do nothing to compromise the chain's safety.</p>

Reviewer: ```suggestion
<p>The <a href="https://developer.arbitrum.io/sequencer">Sequencer</a> is the entity granted specific privileges over ordering transactions; once the Sequencer commits to an ordering (by posting a batch on Ethereum), it has no say over what happens next (i.e., execution). A malicious/faulty Sequencer can do things like reordering transactions or <em>temporarily</em> delaying a transaction's inclusion — things which could be, to be sure, annoying and bad — but can do nothing to compromise the chain's safety.</p>
```

<p>The <em>Validators</em> are the ones responsible for the safety of the chain; i.e., making staked claims about the chain state, disputing each other, etc.</p>

<p>Currently, on Arbitrum One, the Sequencer is a centralized entity maintained by Offchain Labs. Eventually, the single Sequencer will be replaced by a distributed committee of Sequencers who come to consensus on transaction ordering. This upgrade will be an improvement; we don't want you to have to trust us not to reorder your transactions. However, it also isn't <em>strictly</em> necessary for Arbitrum One to achieve its most fundamental properties.</p>

<p>In other words: <em><strong>An Arbitrum Rollup chain with a centralized Sequencer could theoretically still be trustless!</strong></em></p>

<p>Which is to say — the more important thing than decentralizing the Sequencer, i.e., the thing you ought to care more about — is decentralizing the <em>Validators</em>.</p>

> <p>Arbitrum One's Validator set is currently whitelisted; overtime, the whitelist will grow and then be removed entirely. For more info see <a href="https://developer.arbitrum.io/mainnet-beta">"Mainnet Beta"</a>.</p>

Reviewer 2: ```suggestion
<p>Arbitrum One's Validator set is currently allowlisted; overtime, the allowlist will grow and then be removed entirely. For more info see <a href="https://developer.arbitrum.io/mainnet-beta">"Mainnet Beta"</a>.</p>
```

answer to "why?" [here](https://www.notion.so/arbitrum/Why-don-t-we-say-blacklist-or-whitelist-anymore-fbd38356791c41d2a9f1dbbd0c2f919e).

Author: Changed in Notion and regenerated the FAQs 💪

### Why was "one week" chosen for Arbitrum One's dispute window? {#why-was-one-week-chosen-for-arbitrum-ones-dispute-window}

> <p>Generally, some dispute period is necessary for Arbitrum Rollup so that validators have time to dispute an invalid assertion.<br />

Reviewer: ```suggestion
<p>Generally, some amount of time is necessary for the Arbitrum validators to dispute an invalid assertion.<br />
```

<br />
A week is expected to be more than enough time for validators to carry out an interactive dispute, assuming they don't encounter difficulty in getting their transactions included on L1. One week was chosen following the general consensus among the Ethereum research community — as well as other layer 2 projects — to provide enough time for the community to socially coordinate in the case of a coordinated Ethereum-staker censorship attack.</p>

<p></p>

> ### What's the state of Arbitrum One's decentralization? {#whats-the-state-of-arbitrum-ones-decentralization}

Reviewer: Is this still correct with the DAO up and running?
cc @DZGoldman

Author: It was already updated in the hardcoded questions, but not on the CMS. I've updated the entry with the latest answer. Thanks guys !
